content/commands/httl/index.md

```diff
@@ -0,0 +1,90 @@
+---
+acl_categories:
+- '@read'
+- '@hash'
+- '@fast'
+arguments:
+- display_text: key
+  key_spec_index: 0
+  name: key
+  type: key
+- display_text: numfields
+  name: numfields
+  type: integer
+- display_text: field
+  multiple: true
+  name: field
+  type: string
+arity: -4
+categories:
+- docs
+- develop
+- stack
+- oss
+- rs
+- rc
+- oss
+- kubernetes
+- clients
+command_flags:
+- readonly
+- fast
+complexity: O(N) where N is the number of arguments to the command
+description: Returns the TTL in seconds of a hash field.
+group: hash
+hidden: false
+key_specs:
+- RO: true
+  access: true
+  begin_search:
+    spec:
+      index: 1
+    type: index
+  find_keys:
+    spec:
+      keystep: 1
+      lastkey: 0
+      limit: 0
+    type: range
+linkTitle: HTTL
+since: 8.0.0
+summary: Returns the TTL in seconds of a hash field.
+syntax_fmt: HTTL key numfields field [field ...]
+syntax_str: numfields field [field ...]
+title: HTTL
+---
+Returns the remaining TTL (time to live) of a hash key's field(s) that have a set expiration.
+This introspection capability allows a Redis client to check how many seconds a
+given hash field will continue to be part of the hash key.
+
+The return value in case of an error is one of:
```

Comment:
Should remove either "the return value" from this line or "the command returns" from the list items. Probably remove from the list items to be consistent with the other pages?